nncf/torch/functions.py

```diff
@@ -39,10 +39,10 @@ def backward(ctx, grad_output):
 
 class STThreshold(torch.autograd.Function):
     @staticmethod
-    def forward(ctx, input_):
-        output = (input_ > 0.5).type(input_.dtype)
+    def forward(ctx, input_, threshold=0.5):
+        output = (input_ > threshold).type(input_.dtype)
         return output
 
     @staticmethod
     def backward(ctx, grad_output):
-        return grad_output
+        return grad_output, None
```

Comment:
Modify/add to existing unit test to test forward, backward w.r.t. the new input arguments

Reply:
Seems that the two functions in this file (`STRound` and `STThreshold`) do not have any tests now. Where to add them? @ljaljushkin

Reply:
@yujiepan-work I'd suggest adding to `tests/torch/autograd_functions/` folder